internal/access/access_key.go

```diff
@@ -41,6 +41,10 @@ func CreateAccessKey(c *gin.Context, accessKey *models.AccessKey, identityID uid
 		return "", fmt.Errorf("get access key identity: %w", err)
 	}
 
+	if accessKey.Name == "" {
+		accessKey.Name = fmt.Sprintf("%s-%s", identity.Name, time.Now().Format("2006-01-02-15:04:05.000000"))
```

Comment:
This format looks like it is almost `time.RFC3339` except that it uses `-` instead of `T` to separate the date and time, and doesn't have a time zone.

Excluding the timezone seems like a nice way of keeping it more readable, but maybe we should call `.UTC()` on the time first, to make sure the timestamp is in UTC?

I'm also wondering if we should use the `T` to separate the two parts instead of the `-`. 

(Not blocking)